docs/lab/overview/introduction.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: "Introduction"
+---
+
+Vital's lab test API allows digital health companies to carry out at-home lab testing.
+Companies can use the API to order a variety of tests, using various sampling methods.
+
+We partner with CLIA and CAP certified labs across the United States that cover a variety of diagnostic tests. For example lipids panels, metabolic profiles, hormone tests and more.
+We collect these test samples via dried bloodspot, saliva or phlebotomy.
+
+### Features
+
+Our lab test API gives you an end-to-end experience, from ordering a test all the way to receiving results, all with one API call!
+
+What we offer:
+
+* Multiple [test modalities](/lab/overview/testing-modalities) through one unified API.
+* No Physician required - you can use your own physician if you have one, or we will automatically assign one from our network.
+* Updates throughout the test lifecycle - we automatically send you webhook updates and SMS updates to your patients so that you both never miss an update.
+* Test results: we will update you once the test results are ready. In case of abnormal results, we also provide follow-ups through our Physician network.
+* Optional uploading of health insurance card: we support providing your patients health insurance cards for insurance billing.
+* 1:1 support: after production launch, you will have direct access to slack and 1:1 support with our engineers.
+
+### Start integrating
+
+You can sign-up today to our sandbox environment. The [quickstart](/lab/overview/quickstart-api) guide will help you getting from zero to ordering tests in less than 30 minutes! Once the integration is ready, switching to production will just be a matter of switching API keys.
+
+### Production launch
+
+As a first step, we need to define what type of testing you want to offer to your users. We will have an introduction call to go over the test, projected volumes, customizations for kits and other questions you might have. Once we have all this information, we can begin setting up the test kits and distribution. Finally, we can set up your integration and dashboard to start ordering tests via our API.
+
+In [this page](https://tryvital.io/testing), you can check out some examples of the test panels we offer, and book an introductory call with us.
+
+### Support
+
+You will be assisted throughout the integration process with Vital. After launch, you will have a dedicated account manager with access to slack, and 1:1 support with our engineers.
+
+### Coverage
+
+Our lab test API is available throughout the US. This is our current coverage by test modality:
+
+* [At-home test kits](/lab/overview/testing-modalities#at-home-test-kits): 49 states (all excluding NY).
+* [In-lab phlebotomy](/lab/overview/testing-modalities#in-lab-phlebotomy): 49 states (all excluding NY).
```

Comment:
We're using in-lab phlebotomy here but walk_in_tests in our API. We should standardise this to say walk_in_tests

Reply:
Yeah.. Unfortunately I prefer the name `in-lab phlebotomy`, but it's _a lot_ easier to change the docs than the API 😬

Reply:
Its not neccessarily phlebotomy in a lab though, it could be anything